RiotSwiftUI/Modules/Common/Util/BorderedInputFieldStyle.swift

```diff
@@ -51,7 +51,7 @@ struct BorderedInputFieldStyle: TextFieldStyle {
         if (theme.identifier == ThemeIdentifier.dark) {
             return (isEnabled ? theme.colors.primaryContent : theme.colors.tertiaryContent)
         } else {
-            return (isEnabled ? theme.colors.primaryContent : theme.colors.quarterlyContent)
+            return (isEnabled ? theme.colors.primaryContent : theme.colors.quaternaryContent)
```

Comment:
It seems there were different names for the same colour. I have updated SwiftUI to be consistent with compound/DesignKit but left both available for UIKit.